src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -667,7 +667,18 @@ public static void Initialize(Collection<Exception> errors, List<string> moduleP
             }
             else
             {
-                var systemResourceRoot = Path.Combine(Platform.GetFolderPath(Environment.SpecialFolder.System), "Configuration");
+                // DSC SxS scenario
+                var psAssembly = Assembly.GetEntryAssembly();
+                var configSystemPath = Path.GetDirectoryName(psAssembly.Location);
```

Comment:
We could use `Utils.DefaultPowerShellAppBase`.

Reply:
For which path?

Reply:
Here you get PowerShell home directory - `Utils.DefaultPowerShellAppBase` returns the same.

Reply:
@mbreakey3 Are you going to make this change?

Reply:
Yes - the change should be there